pyproject.toml

```diff
@@ -20,7 +20,7 @@ classifiers = [
         'Environment :: Console',
         'Intended Audience :: Developers',
         'Intended Audience :: Science/Research',
-        'License :: OSI Approved :: BSD License',
+        'License :: Other/Proprietary License',
```

Comment:
https://pypi.org/classifiers/

Using the convention from above.

Reply:
this is not proprietary.  Is there one that says something like free for use with restrictions or sthg that actually matches what hte license says?

Reply:
`License :: Free To Use But Restricted`

How about this?

Reply:
Please see the new update.

`License :: Free To Use But Restricted` is used.